src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -192,14 +168,40 @@ public double LightElevationDegrees
             set { lightElevationDegrees = value; }
         }
 
-#if DEBUG
-        /// <summary>
-        /// The TestSelectionCommand is used in the WatchSettingsControl
-        /// to test the ability to toggle a boolean effect variable
-        /// representing the selection state.
-        /// </summary>
-        public Dynamo.UI.Commands.DelegateCommand TestSelectionCommand { get; set; }
-#endif
+        private Dictionary<string, Model3D> geometryDictionary= new Dictionary<string, Model3D>();
+
+        public Dictionary<string, Model3D> GeometryDictionary
+        {
+            get
+            {
+                return geometryDictionary;
+            }
+
+            set
+            {
+                geometryDictionary = value;
+            }
+        }
+
+        public List<Model3D> GeometryValues
+        {
+            get
+            {
+                return GeometryDictionary == null ? new List<Model3D>() :
+                   GeometryDictionary.Select(x => x.Value).ToList();
+            }
+        }
+
+        public List<Model3D> GeometryValuesWithoutConstants
```

Comment:
@ramramps Similar to above, let's make these private members unless required to be visible externally or to data binding.